.circleci/config.yml

```diff
@@ -16,11 +16,21 @@ version: 2.1
 
 commands:
   checkout_with_submodules:
+    parameters:
+      ethereum_tests:
+        description: Include a heavy Ethereum tests submodule if needed.
+        type: boolean
+        default: true
     steps:
       - checkout
       - run:
           name: "Update submodules"
           command: |
+            if [[ <<parameters.ethereum_tests>> == false ]]
+            then
+              git config submodule.tests.update none
+            fi
```

Comment:
Perhaps you can also unconditionally disable checkout of the `LegacyTests` submodule of Ethereum tests. We don't use those.